docs/tutorials-samples/targeting-core-for-new-libraries.md

```diff
@@ -0,0 +1,298 @@
+# Targeting with .NET Core Libraries
+
+Targeting .NET Core for new libraries can be done a number of ways, depending on the platform(s) you wish to support.
+
+## How do I target .NET Core?
+
+First things first:
```

Comment:
Do we want to add something about prerequisites, such as what you will need to install before following the instructions in this document, even if it's just a pointer to another document?